src/Flows.js

```diff
@@ -1090,12 +1094,38 @@ export class Flow extends PureComponent {
           })
           .catch(failed);
       } else if (this.state.mode === 'attention') {
+        let use_search = !!this.state.search_param;
+        let use_regex = use_search && !!this.state.search_param.match('/.+/');
```

Comment:
```suggestion
        let use_regex = use_search && !!this.state.search_param.match(new RegExp('/.+/'));
```

explicit is better than implicit :)

(其实是第一眼没看懂是什么操作)

Reply:
用字面量就好了
```js
!!this.state.search_param.match(/.+/);
```

Reply:
`new RegExp('/.+/')` 的字面量不是 `/.+/` 吧. 还要 escape `/`